android_app/app/src/main/java/com/health/openscale/gui/views/Caliper1MeasurementView.java

```diff
@@ -0,0 +1,72 @@
+/* Copyright (C) 2018  olie.xdev <olie.xdev@googlemail.com>
+*
+*    This program is free software: you can redistribute it and/or modify
+*    it under the terms of the GNU General Public License as published by
+*    the Free Software Foundation, either version 3 of the License, or
+*    (at your option) any later version.
+*
+*    This program is distributed in the hope that it will be useful,
+*    but WITHOUT ANY WARRANTY; without even the implied warranty of
+*    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*    GNU General Public License for more details.
+*
+*    You should have received a copy of the GNU General Public License
+*    along with this program.  If not, see <http://www.gnu.org/licenses/>
+*/
+package com.health.openscale.gui.views;
+
+import android.content.Context;
+import android.graphics.Color;
+
+import com.health.openscale.R;
+import com.health.openscale.core.datatypes.ScaleMeasurement;
+import com.health.openscale.core.evaluation.EvaluationResult;
+import com.health.openscale.core.evaluation.EvaluationSheet;
+
+public class Caliper1MeasurementView extends FloatMeasurementView {
+    // Don't change key value, it may be stored persistent in preferences
+    public static final String KEY = "caliper1";
+
+    public Caliper1MeasurementView(Context context) {
+        super(context, R.string.label_caliper1_female, R.drawable.ic_caliper1);
+
+        if (getScaleUser().getGender().isMale()) {
+            setNameView(getResources().getText(R.string.label_caliper1_male));
+        }
+    }
+
+    @Override
+    public String getKey() {
+        return KEY;
+    }
+
+    @Override
+    protected float getMeasurementValue(ScaleMeasurement measurement) {
+        return measurement.getCaliper1();
+    }
+
+    @Override
+    protected void setMeasurementValue(float value, ScaleMeasurement measurement) {
+        measurement.setCaliper1(value);
+    }
+
+    @Override
+    public String getUnit() {
+        return "mm";
```

Comment:
Shouldn't this be presented in cm? Feels a bit too exact to enter a mm value with two decimals.